tests/5.0/target/test_target_device.c

```diff
@@ -13,28 +13,33 @@
 #include <omp.h>
 #include "ompvv.h"
 
-#define N 1028
+#define N 1024
 
 // Required for 'device(ancestor: 1)
 #pragma omp requires reverse_offload
 
 int test_target_device_ancestor() {
 
-    int i, which_device;
+    int which_device;
     int a[N];
     int errors = 0; 
+    int is_shared_env = 0;
+    which_device = 0;
 
     for (int i = 0; i < N; i++) {
         a[i] = i;
     }
 
     OMPVV_TEST_AND_SET(errors, omp_get_num_devices() <= 0);
-    OMPVV_ERROR_IF(omp_get_num_devices() <= 0, "Since no target devices were found, this test"
+    OMPVV_WARNING_IF(omp_get_num_devices() <= 0, "[SKIPPED] Since no target devices were found, this test"
```

Comment:
Same remark as above - it will not be skipped, but it won't test as much and it also won't pass. _(As the host is a shared-memory device.)_

Reply:
Updated.

Reply:
Changed the warning message with the following:
"[WARNING] may not be able to detect errors since the target system does not have a target device."